### ISSUE TITLE
lsr_role2collection.py does not convert the legacy role path if the role is not the one to be converted.

### ISSUE THREAD

**Issue by nhosoi** [linux-system-roles/auto-maintenance#65]
This is an example of README of sap-hana-preconfigure, in which some roles are referred.
```
./README.md:- 'linux-system-roles.timesync'
./README.md:- 'linux-system-roles.sap-base-settings' (for RHEL 7.x until RHEL 7.5)
./README.md:- 'linux-system-roles.sap-preconfigure' (for RHEL 7.6 and later and RHEL 8.x)
./README.md:        - { role: linux-system-roles.sap-base-settings }
./README.md:        - { role: linux-system-roles.sap-hana-preconfigure }
```
The corresponding converted result:
```
./roles/sap-hana-preconfigure/README.md:- 'linux-system-roles.timesync'
./roles/sap-hana-preconfigure/README.md:- 'linux-system-roles.sap-base-settings' (for RHEL 7.x until RHEL 7.5)
./roles/sap-hana-preconfigure/README.md:- 'linux-system-roles.sap-preconfigure' (for RHEL 7.6 and later and RHEL 8.x)
./roles/sap-hana-preconfigure/README.md:        - { role: linux-system-roles.sap-base-settings }
./roles/sap-hana-preconfigure/README.md:        - { role: sap.rhel.sap-hana-preconfigure }
```
All the roles must be converted to the collection format, but it'd be hard to automate it. The FQCNs need to be provided since, e.g., in this example, linux-system-roles.timesync could be either fedora.linux_system_roles.timesync or redhat.rhel_system_roles.timesync, and the others would be sap.rhel.sap-{base-settings,preconfigure}.

**Comment by spetrosi** [comment]
@nhosoi was this issue resolved by #68?

**Comment by nhosoi** [comment]
> @nhosoi was this issue resolved by #68?

Yep, thanks for the reminder, @spetrosi. Closing this issue.
